docs/docs/adding-markdown-pages.md

```diff
@@ -5,7 +5,7 @@ title: Adding Markdown Pages
 Gatsby can use markdown files to create pages in your site.
 You add plugins to read and understand folders with markdown files and from them create pages automatically.
 
-Here's the steps Gatsby follows for making this happen.
+Here are the steps Gatsby follows for making this happen.
```

Comment:
I love this change! If you can, can you find more instances of these? I'm sure you'll find a few more!